kernel/src/process.rs

```diff
@@ -1510,17 +1511,17 @@ fn exceeded_check(size: usize, allocated: usize) -> &'static str {
 }
 
 impl<C: 'static + Chip> Process<'_, C> {
+    #[inline(always)]
```

Comment:
What does this do?

Reply:
Oh, sorry, that's left over from when I was trying a different way, you shouldn't need it.